pkg/trace/writer/stats.go

```diff
@@ -238,6 +239,10 @@ func (w *StatsWriter) resolveContainerTags(p *pb.ClientStatsPayload) {
 		p.Tags = nil
 		return
 	}
+	if p.ContainerID != "" && p.Tags != nil {
```

Comment:
I don't see a scenario where we need to append Tags to already present ones.

Reply:
It could happen that the container tags become stale?

Today we don't add container tags upstream in the concentrator or before it. But if we did, then by the time the stats come out (10 sec) perhaps the container tags mapping will have changed in some way?

We might need to change this code again once we properly support adding container tags upstream in the concentrator. But for right now I think this change is safe.